Taskfile.yml

```diff
@@ -261,6 +261,17 @@ tasks:
         -target-backend=mongodb
         -no-xfail={{.NO_XFAIL}}
 
+  convert-test-results:
```

Comment:
Do we even need this Taskfile target, given that we don't use it for unit tests?